### ISSUE TITLE
Focus state does not meet WCAG 2.2

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-design-system#325]
## What
> Our current focus state does not meet WCAG 2.2 AA/AAA proposed criteria. We should aim for AAA on this criteria.

## Why
>- Focus states help users to see where they are on the page
>- Currently we have a purple box around the focused element
><img width="250" alt="Screenshot 2020-08-21 at 09 02 24" src="https://user-images.githubusercontent.com/10854265/90867275-0b6fed80-e38d-11ea-93e0-b3f04bf886e8.png">
>- We need to make this better while we have the opportunity, we should aim for WCAG 2.2 AAA
>- Criteria: 3px border around whole control, change of contrast is at least 4.5:1 different from unfocused (i.e. there is a colour background within the focus background)

## Anything else
> [GOV.UK focus states](https://design-system.service.gov.uk/get-started/focus-states/) replicated to [NHS Service Manual](https://service-manual.nhs.uk/design-system/styles/focus-state)
> ![image](https://user-images.githubusercontent.com/10854265/90867639-9e108c80-e38d-11ea-82ff-2a24655ff936.png)
> ![image](https://user-images.githubusercontent.com/10854265/90867667-a799f480-e38d-11ea-8de4-2d4cbcc96434.png)

**Comment by KaterinaKir** [comment]
New focus state styling: Yellow fill (#ffdd00'), purple outline (#3c1053), 3px, 

Focus styling overrides original style of the element, e.g. blue button and white text, becomes yellow (fill) and purple (outline and text) when keyboard operated focus is activated.
<img width="237" alt="Screenshot 2020-08-25 at 14 37 34" src="https://user-images.githubusercontent.com/57525718/91182308-e9a29d80-e6e1-11ea-9f6f-6d11d8b53aa6.png">

**Comment by KaterinaKir** [comment]
<img width="408" alt="Screenshot 2020-08-26 at 09 00 00" src="https://user-images.githubusercontent.com/57525718/91277307-8ca6f600-e77a-11ea-9111-551b78f28e82.png">

When user selects button with a keyboard, focus state is activated and it overrides original styling. E.g. is user clicks on purple or blue button, it will turn yellow with outline (3px)

**Comment by daylesalmon** [comment]
This is a fairly big change that will involve going through every component in the design system and adding a custom focus state to each.

A new class called "wmnds-bg-yellow" will need to be added to the design system and anything that is a child of this class will also need to have a custom focus state too.

So in short...

All components need 2 new classes/styling added:
- One for generic focus state
- Another for wmnds-yellow *:focus state
